src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
```

Comment:
According to the checklist, it's not recommended to use tag names for styling, except for `html` and `body`. However, setting global styles like this can have unintended side effects. It would be better to add a class to the body and style it instead. This way, you ensure that the styles are only applied to the intended elements.